### ISSUE TITLE
Package for idea box interface

### ISSUE THREAD

**Issue by quakerquickoats** [rememberYou/emacs-ideas#2]
## Description

This package aims to add some functionality for this idea box to GNU Emacs. We would be able to add ideas in markdown, list them, edit them and comment on them; as well as being able to provide links to local info files and links to (built-in) source code, and possibly example config code. All from our familiar environment.

**Comment by rememberYou** [comment]
That's a great idea. This will allow new ideas to be consulted regularly without having to visit the repository first.

**Comment by quakerquickoats** [comment]
Perhaps it could be just a magit shortcut, to the repo, which I think may do issues and such? I am not familiar with it myself

**Comment by alphapapa** [comment]
@quakerquickoats You're essentially talking about Worg.

**Comment by thblt** [comment]
I'd be surprized if there wasn't a package for Github issues already?  Magithub maybe?

**Comment by quakerquickoats** [comment]
@alphapapa I do not think that I am essentially talking about Worg. That appears to be an org-mode specific community website for org-mode things. Aside from https://orgmode.org/worg/org-devel.html - there seems to be nothing related to packages, idea-generation, or collaboration on development on said ideas at all. Perhaps you misunderstand the purpose of this emacs idea box? Feel free to comment for clarifications.

**Comment by alphapapa** [comment]
The point was that what you described would work the way Worg does.  You could imitate it instead of starting from scratch.
